src/Vision/testpi.py

```diff
@@ -300,7 +300,7 @@ def normalizeImage(pairMidpoint):
     fullPairMidpoint = (pairPointOrient / imageMidpoint)
     # prints after ewww math THIS IS THE CONTOUR MIDPOINT from...
     # Contour Midpoint Scale (-1 to 1)
-    return fullPairMidpoint
+    return fullPairMidpoint 
```

Comment:
You have an extra space at the end of this

Reply:
![](http://4.bp.blogspot.com/-i9gjbiGZZF4/TzGWsVZnYuI/AAAAAAAAEws/MqPWmNUCfg4/s1600/spaaace.jpg)